scripts/checks/pragma-validity.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
```

Comment:
Could we add a note on all unedited copied files, something like:
```suggestion
// TODO: Replace file content with symlink to ./lib/openzeppelin-contracts/scripts/ when available.
```